### PR TITLE
feat: add call support to verified requests for prover

### PR DESCRIPTION
**Pull request by nazarhussain**
**Motivation**

Make the prover more feature rich. 

**Description**

Add support for eth **call** method. 

Partially #4706

**Steps to test or reproduce**

- Run all tests.

### CODE REVIEW

File: packages/prover/scripts/generate_fixtures.ts [hunk 1]
```diff
@@ -23,41 +23,104 @@ const networkURLs: Record<NETWORK, {beacon: string; rpc: string}> = {
...
-    .data as Record<string, unknown>;
+async function rawEth(
+  network: NETWORK,
+  method: string,
+  params: unknown[]
+): Promise<{payload: Record<string, unknown>; response: Record<string, unknown>}> {
+  const payload = {id: idIndex++, jsonrpc: "2.0", method, params};
+
+  const response = (
+    await axios({
```
Comment (dapplion) [inline packages/prover/scripts/generate_fixtures.ts:34]: Why axios instead of native fetch? Will be supported by default in NodeJS too soon

**Review by dapplion — COMMENTED** [review]
@nazarhussain How does the VM instance resolve SLOAD SSTORE opcodes?

File: packages/prover/scripts/generate_fixtures.ts (hunk 1 above)
Comment (nazarhussain) [reply packages/prover/scripts/generate_fixtures.ts:34]: It was already used in other packages for e2e tests so used here for consistent pattern.

**Comment by nazarhussain** [comment]
@dapplion It have internal `state manager` which is storing references of accessed storages. 

https://github.com/ethereumjs/ethereumjs-monorepo/blob/11a5ab8d947923619eabbd0ab591478409f22762/packages/vm/src/eei/vmState.ts#L165
https://github.com/ethereumjs/ethereumjs-monorepo/blob/11a5ab8d947923619eabbd0ab591478409f22762/packages/vm/src/eei/vmState.ts#L169

And in interpreter they resolve the opcode with following execution. 
https://github.com/ethereumjs/ethereumjs-monorepo/blob/11a5ab8d947923619eabbd0ab591478409f22762/packages/evm/src/opcodes/functions.ts#L685-L694

**Review by dapplion — APPROVED** [review]
Looks great!

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.9.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.9.0) :tada:

**Pull request merged**